### PR TITLE
Updated Russian translation for the actual version

### PR DESCRIPTION
**Pull request by ArionWT**
Adapted for [Roboto Condensed Regular](https://fonts.google.com/specimen/Roboto+Condensed) font.

### CODE REVIEW

File: Resources/Locales/ru/openspades.po [hunk 1]
```diff
@@ -313,9 +313,9 @@ msgid "Unknown Reason"
 msgstr "Неизвестная причина"
 
 #: Sources/Client/ScoreboardView.cpp:331
-#msgctxt "Client"
-#msgid "Spectator{1}"
-#msgid_plural "Spectators{1}"
+msgctxt "Client"
+msgid "Spectator{1}"
+msgid_plural "Spectators{1}"
 #msgstr[0] "Наблюдатель{1}"
```
Comment (ArionWT) [inline Resources/Locales/ru/openspades.po]: Remove #

**Comment by ArionWT** [comment]
If #507 fixed, strings 319, 320, 321 should be changed to:
```
 msgstr[0] "Наблюдатель{1}"
 msgstr[1] "Наблюдатели{1}"	
 msgstr[2] "Наблюдатели{1}"
```

File: Resources/Locales/ru/openspades.po [hunk 2]
```diff
@@ -313,9 +313,9 @@ msgid "Unknown Reason"
...
 #msgstr[1] "Наблюдателей{1}"
```
Comment (ArionWT) [inline Resources/Locales/ru/openspades.po]: Remove # and change to msgstr[1] "Наблюдатели{1}"

File: Resources/Locales/ru/openspades.po [hunk 3]
```diff
@@ -313,9 +313,9 @@ msgid "Unknown Reason"
...
 #msgstr[2] "Наблюдателей{1}"
```
Comment (ArionWT) [inline Resources/Locales/ru/openspades.po]: Remove # and change to msgstr[3] "Наблюдатели{1}"

File: Resources/Locales/ru/openspades.po (hunk 1 above)
Comment (ArionWT) [inline Resources/Locales/ru/openspades.po]: Remove #

File: Resources/Locales/ru/openspades.po (hunk 2 above)
Comment (ArionWT) [inline Resources/Locales/ru/openspades.po]: Remove # and change to msgstr[1] "Наблюдатели{1}"

File: Resources/Locales/ru/openspades.po (hunk 3 above)
Comment (ArionWT) [inline Resources/Locales/ru/openspades.po]: Remove # and change to msgstr[2] "Наблюдатели{1}"

File: Resources/Locales/ru/openspades.po [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 "Project-Id-Version: OpenSpades\n"
 "Report-Msgid-Bugs-To: i at yvt.jp\n"
-"POT-Creation-Date: 2016-12-25 23:42-0200\n"
-"PO-Revision-Date: 2016-12-25 23:42-0200\n"
-"Last-Translator: ArionWT <arionwhiteac@gmail.com>\n"
-"Language-Team: \n"
+"POT-Creation-Date: 2016-12-25 23:47-0200\n"
+"PO-Revision-Date: 2017-01-03 11:59+0300\n"
+"Last-Translator: ArionWT\n"
+"Language-Team: ArionWT, noway421, feikname, CenturionFox, VelocityRa, DarkDracoon\n"
```
Comment (yvt) [inline Resources/Locales/ru/openspades.po]: [`Language-Team` should be the English name of the language](https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html).

**Comment by yvt** [comment]
As your suggestion I tried Roboto Condensed Regular. It did look better than Google Noto Fonts, but it still looked too light, and the bold variant on the other hand seemed too bold.

So I made a semi-bold version from the regular variant. Does this look good to you? It can be downloaded [here](https://github.com/yvt/openspades-paks/blob/master/OfficialMods/NotoFonts.pak), if you're interested.

Original Roboto Condensed Regular:
roboto-0

Roboto Condensed Regular emboldened by 25 em units:
roboto-25

**Comment by noway421** [comment]
Emboldened version looks way better, not sure if it's worth the maintenance of that extra font file, but I'd choose second version.

**Comment by ArionWT** [comment]
Semi-bold version of the font looks much better.

**Comment by yvt** [comment]
LGTM :+1:

**Pull request merged**